tests/test_auxtel_standby_atcs.py

```diff
@@ -38,9 +38,19 @@ class TestStandbyATCS(
     async def basic_make_script(self, index):
         self.script = StandbyATCS(index=index)
         self.atcs_mock = ATCSMock()
+        self.script._atcs.disable_checks_for_components = unittest.mock.Mock()
```

Comment:
Same comment as above. This unit test is using `ATCSMock` any reason for changing it?